docs/dev/dev_setup.rst

```diff
@@ -203,11 +203,97 @@ with linters and other code analysis tools.
 Creating a Workspace
 -------------------------------------------------------------------------------
 
-PUDL needs to know where to store its big piles of inputs and outputs. It also comes
-with some example configuration files. The ``pudl_setup`` script lets PUDL know where
-all this stuff should go. We call this a "PUDL workspace". The script will not create
-a new directory based on your arguemnts, so make sure whatever directory path you
-pass as <PUDL_DIR> already exists.
+PUDL Workspace Setup (new method)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. note::
+
+    If you used ``pudl_setup`` to setup your pudl workspace,
+    skip ahead to :ref:`Old PUDL Setup`
+
+PUDL needs to know where to store its big piles of inputs and outputs.
+The ``PUDL_OUTPUT`` and ``PUDL_CACHE`` environment variables lets PUDL know where
+all this stuff should go. We call this a "PUDL workspace".
+
+First, create a directory to store local caches of raw PUDL data. Then set
+``PUDL_CACHE`` variable to the path of the new directory:
```

Comment:
Maybe say "Then create an environment variable called `PUDL_CACHE` to store the path to this new directory.